logexporter/cmd/Dockerfile

```diff
@@ -15,12 +15,10 @@
 # This file builds an image for the log exporter tool. For more info,
 # have a look at the tool's README.md file.
 
-FROM fedora
+FROM python:2.7-alpine
```

Comment:
Have you tested that this works? I remember I was facing many issues with smaller base images and had to eventually settle on fedora.

Reply:
Yes, I ran this container on my machine and was able to run gsutil, main go binary and gcloud.

Reply:
Ok.